addons/web/static/src/search/search_model.js

```diff
@@ -913,6 +917,9 @@ export class SearchModel extends EventBus {
         const index = this.query.findIndex((queryElem) => queryElem.searchItemId === searchItemId);
         if (index >= 0) {
             this.query.splice(index, 1);
+            if (searchItem.type === "groupBy") {
```

Comment:
why? If you have 2 groupbys active and you unselect one, orderByCount should remain true